config/samples/csi_v1_hostdefinition.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: csi.ibm.com/v1
+kind: HostDefinition
+metadata:
+  name: ibm-block-csi
+  namespace: default
+  labels:
+    app.kubernetes.io/name: ibm-block-csi
+    app.kubernetes.io/instance: ibm-block-csi
+    app.kubernetes.io/managed-by: ibm-block-csi-operator
+    release: v1.10.0
+spec:
+  hostDefinition:
+    prefix: ibm_
+    connectivity: iscsi
```

Comment:
maybe it's better if the default will be fc?
ii believe it's the common case..

Reply:
IMO the default should [not](https://github.com/IBM/ibm-block-csi-operator/pull/256#discussion_r874494517) be any specific "global" connectivity

Reply:
the default should be empty?

Reply:
sure

Reply:
done, what is the default prefix that we add to volumes? I will change the prefix value to it

Reply:
the only non-empty default volume prefix we have is in [svc](https://github.com/IBM/ibm-block-csi-driver/blob/v1.9.0/controller/array_action/array_mediator_svc.py#L190) (note that the underscore delimiter cannot be changed by the user), and only because the orchestrator is allowed to request a name that starts with a digit, which is not allowed in svc

Reply:
so e.g if someone creates volume at ds8k storage it won't add any characters to the volume name if the user won't add a prefix in the storage_class?

Reply:
exactly

Reply:
@ArbelNathan @oriyarde do we want any default prefix? leave it empty in this YAML file?

Reply:
maybe:
`# connectivity: iscsi/ fc/ nvme`? (commented)

Reply:
I don't think the connectivity type is a good prefix.
I would leave it empty, no default

Reply:
done